packages/compass-schema/src/components/coordinates-minichart/utils.js

```diff
@@ -46,7 +48,7 @@ const getHereAttributionMessage = async function (bounds, level) {
   });
 
   const copyrightString = copyrights.join(', ');
-  return ` &copy; 1987-2019 HERE${
+  return ` &copy; 1987-${thisYear} HERE${
     copyrightString.length > 0 ? `, ${copyrightString}` : ''
   } | <a href="https://legal.here.com/en/terms/serviceterms/us">Terms of Use</a>`;
```

Comment:
We probably want this too

```suggestion
  } | <a target="_blank" href="https://legal.here.com/en/terms/serviceterms/us">Terms of Use</a>`;
```